specification/mysql/resource-manager/Microsoft.DBforMySQL/FlexibleServers/PrivateEndpointConnection.tsp

```diff
@@ -0,0 +1,147 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+import "./Server.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.DBforMySQL;
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-interface-requires-decorator" "Fixme: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@tag("PrivateEndpointConnections")
+interface PrivateEndpointConnections {
```

Comment:
for this operation group, use customized code as if we use arm template, it requires a resource model, but if we use the name `PrivateEndpointConnection` as the resource model ,it will duplicate with `Azure.ResourceManager.CommonTypes.PrivateEndpointConnection `

Reply:
we can't use non-arm templates